README.md

```diff
@@ -4,55 +4,29 @@
+3. Import the project into Android Studio
 
 When building for the first time, gradle will download all dependencies so it'll take a few minutes to complete. Subsequent builds will be faster.
 
-Once the APK is generated, there are two ways for installing it:
-
-## Installing
-
-#### a) By manually transfer the binary to a phone
-
-Android by default blocks installation of apps from outside the Play Store. To enable it, go to phone `Settings > Security` and enable `Unknown sources`. If you don't have this setting (which is the case starting from Android Oreo), your phone should automatically open this setting when you try installing the app. Once that's done, just send the APK to your phone through email, Dropbox or any other medium of your choice.
+## Running the app
 
-#### b) Over a USB cable
-
-This setup requires some initial setup, but is recommended for multiple installs. To do so, `USB debugging` needs to be enabled in phone settings. The steps for finding this setting varies with manufacturers.
+An APK can be built and installed to a device directly from Android Studio. The device could be an emulator or a real device. Emulators come pre-configured to run binaries, but running on a real device requires enabling `USB debugging` in phone settings. The steps for finding this setting varies with manufacturers, but should be somewhat along the lines of,
 
 - If your phone settings has a search option, try searching for `Build number`.
 - If search is unavailable, try navigating to `Settings > System > About phone`. Scroll to the bottom to find `Build number`.
 - Tap on `Build number` for 5 times until you see a message saying "You are now a developer!".
 - Go back to phone settings. A new setting group called  `Developer options `will now be available.
 - Open `Developer options` and enable `USB debugging`.
 
-The binary can now be installed on the phone by running:
+If `adb` fails with a `no devices/emulators found` error, it is possible that the device is connected to the computer in charging only mode. In this case, you should see a notification on the device to change this to debugging.
 
-```
-~/Library/Android/sdk/platform-tools/adb install {path to apk}
-```
+## Downloading a build from CI
 
-If `adb` fails with a `no devices/emulators found` error, it is possible that the device is connected to the computer in charging only mode. In this case, you should see a notification on the device to change this to debugging.
+We use Bitrise for continuous integration. It is configured to build a binary every time a commit is pushed to the repository. These binaries can be found by navigating [to this page](https://www.bitrise.io/app/db9b195f645cfed7#/builds_) and opening the details of any build. The details page will contain a generated APK called `app-debug.apk` under the "Apps & Artifacts" tab.
 
 ## Build and deploy Simple Server
```

Comment:
We should clarify that the server is not _really_ needed to run the app, and is only needed to sync data across devices.